sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/TypeCoercionSuite.scala

```diff
@@ -31,75 +31,49 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.types._
 import org.apache.spark.util.Utils
 
-class TypeCoercionSuite extends AnalysisTest {
+abstract class TypeCoercionSuiteBase extends AnalysisTest {
```

Comment:
so the change is just code move around? We move tests from `AnsiTypeCoercionSuite` to this case suite?

Reply:
I just moved the cases from `TypeCoercionSuite` to `TypeCoercionSuiteBase`, since some of them work for the ANSI one now.